app/components/atoms/Container/tests/__snapshots__/index.test.js.snap

```diff
@@ -1,36 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`<Container /> should render and match the snapshot 1`] = `
-<View
-  pointerEvents="box-none"
-  style={
-    Object {
-      "flex": 1,
-    }
-  }
->
-  <View
-    collapsable={true}
-    pointerEvents="box-none"
-    style={
-      Object {
-        "flex": 1,
-      }
-    }
-  >
-    <View
-      style={
-        Array [
-          Object {
-            "display": "flex",
-            "flexBasis": 0,
-            "flexGrow": 1,
-            "flexShrink": 1,
-          },
-        ]
-      }
-      testID="container"
-    />
-  </View>
-</View>
-`;
+exports[`<Container /> should render and match the snapshot 1`] = `undefined`;
```

Comment:
Why all components snapshot is the same?

Reply:
Addressed this issue. Thanks for pointing it out.